ExampleCWPlugin.csproj

```diff
@@ -10,10 +10,19 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <Reference Include="$(CWDir)\Content Warning_Data\Managed\*.dll" Private="false"/>
+        <Reference Include="$(CWDir)\Content Warning_Data\Managed\*.dll" Private="false" Publicize="True"/>
     </ItemGroup>
 
-    <Target Name="CopyOutputFileToGame" AfterTargets="Build">
+    <!-- Use BepInEx's Assembly Publicizer to tell the compiler & IDE that every field, method, etc. is public, in the game assemblies. -->
+    <!-- This makes it easier to change fields and call methods that would otherwise be private. -->
+    <ItemGroup>
+      <PackageReference Include="BepInEx.AssemblyPublicizer.MSBuild" Version="0.4.2">
+        <PrivateAssets>all</PrivateAssets>
+        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      </PackageReference>
+    </ItemGroup>
+
+    <Target Name="CopyOutputFileToGame" AfterTargets="AfterBuild">

```

Comment:
is there a reason you changed this from `AfterTargets="Build"` to `AfterTargets="AfterBuild"`?

Reply:
Yes, for some reason Build would not trigger the target. All the times I've used Targets to copy something, I use AfterBuild and that has always worked.

Maybe it's IDE dependant? Hopefully not. Either way, this change was made just to fix this problem. :)

Reply:
From what I can tell, [Build is now the preference instead of BeforeBuild/AfterBuild](https://github.com/dotnet/msbuild/issues/1680). Both Rider and `dotnet build` invoke the target for me. Probably best to go with the recommended style and fix whatever's broken with your build system!

Reply:
> From what I can tell, [Build is now the preference instead of BeforeBuild/AfterBuild](https://github.com/dotnet/msbuild/issues/1680). Both Rider and `dotnet build` invoke the target for me. Probably best to go with the recommended style and fix whatever's broken with your build system!

Oh, weird.... I use Rider too and I still had the problem. I'll try again in a minute with it set to Build :)

Reply:
Yep, that worked!